build.gradle

```diff
@@ -366,66 +350,60 @@ signing {
  * For releasing to your local maven repo, use gradle install
  */
 uploadArchives {
-    if(project.hasProperty('sonatypeUsername') && project.hasProperty('sonatypePassword')){
-        repositories {
-            mavenDeployer {
-                beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+    doFirst {
+        println "Attempting to upload version:$version"
+    }
+    repositories {
+        mavenDeployer {
+            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-                repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                    authentication(userName: sonatypeUsername, password: sonatypePassword)
-                }
+            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                authentication(userName: project.findProperty("sonatypeUsername"), password: project.findProperty("sonatypePassword"))
+            }
 
-                snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots") {
-                    authentication(userName: sonatypeUsername, password: sonatypePassword)
-                }
+            snapshotRepository(url: "https://artifactory.broadinstitute.org/artifactory/libs-snapshot-local/") {
+                authentication(userName: System.env.ARTIFACTORY_USERNAME, password: System.env.ARTIFACTORY_PASSWORD)
```

Comment:
I can change them now if you want... I guess there isn't really any reason to keep them this way.  I just didn't want to have to wait for travis again since it's getting late, but If I'm making other changes I can do these too.